tests/tests.js

```diff
@@ -36,10 +160,21 @@ describe('calculateProductPrice', () => {
 
     expect(price).to.equal(22.04)
   })
+  it('calls the calculateCommuterPrice, getEmployerContribution, and formatPrice functions once for eact of the commuter products', () => {
```

Comment:
This test is good but definitely fails since your function doesn't call either `getEmployerContribution` or `formatPrice` for commuter products. That should have been a hint that you should be calling those functions in that case.